internal/installcfg/installcfg.go

```diff
@@ -248,9 +248,19 @@ func setBMPlatformInstallconfig(log logrus.FieldLogger, cluster *common.Cluster,
 		}
 		yamlHostIdx += 1
 	}
+
+	enableMetal3Provisioning, err := common.VersionGreaterOrEqual(cluster.Cluster.OpenshiftVersion, "4.7")
+	if err != nil {
+		return err
+	}
+	provNetwork := "Unmanaged"
+	if enableMetal3Provisioning {
+		provNetwork = "Disabled"
```

Comment:
actualy the disabled is here. seems that in 4.6 it was unmanaged (or actually meaningless). In 4.7, it should be disabled. So it's fine
@stbenjam

Reply:
It is meaningless, which is why I was asking. If it could it would be nice to remove it and just always use disabled. The signal there's no provisioning network is the missing CR.